packages/desktop-ui-lib/src/lib/time-tracker/timer-session/timer-session.component.html

```
@@ -0,0 +1,102 @@
+<div class="wrap">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)